pkg/compiler/backend/simpledocker.go

```diff
@@ -237,7 +237,7 @@ func (*SimpleDocker) ExtractRootfs(opts compiler.CompilerBackendOptions, keepPer
 // ]
 // Changes uses container-diff (https://github.com/GoogleContainerTools/container-diff) for retrieving out layer diffs
 func (*SimpleDocker) Changes(fromImage, toImage string) ([]compiler.ArtifactLayer, error) {
```

Comment:
Probably the same applies for the img backend

Reply:
> looks good! wdyt about adding a simple (integration) test?

Not so easy... tests are executed in parallel and if I clean up the directory I think that I broke other tests. I will integrate only the test to create a temp file and a temporary dir.

Reply:
Integration tests aren't run in parallel, iirc not even the unit tests